kafka-lib_0_10/src/main/scala/Consumer.scala

```diff
@@ -6,6 +6,16 @@ import scala.util.Try
 trait Consumer[T] {
 
   def kafkaConsumer: KafkaConsumer[String, String]
+  
+  def processBatch(
```

Comment:
If there is no implementation for those methods on this trait, classes with this trait but without a proper implementation for these methods won't compile?